src/klein/interfaces.py

```diff
@@ -22,61 +22,61 @@
 )
 
 if TYPE_CHECKING:  # pragma: no cover
```

Comment:
Most of this diff is sorting the file for later sanity.

We do remove `import IRequestLifecycleT as _IRequestLifecycleT` and replace that with similar handling of other interfaces for consistency.